src/core/opamProcess.mli

```diff
@@ -214,3 +214,6 @@ type 'a job = 'a Job.Op.job
 (**/**)
 val set_resolve_command :
   (?env:string array -> ?dir:string -> string -> string option) -> unit
+
+(** Like Unix.create_process_env, but with correct escaping of arguments on cygwin *)
```

Comment:
```suggestion
(** Like Unix.create_process_env, but with correct escaping of arguments when invoking cygwin executables *)
```
It's a very weird distinction - in the Cygwin build of opam, everything is fine and Unixy; in the _native Windows_ build of opam, all this weirdness comes into play when a native executable calls a Cygwin executable!